tests/unit/Storefront/Pagelet/Header/HeaderPageletLoaderTest.php

```diff
@@ -77,8 +78,10 @@ public function testLoad(): void
         $headerPageletLoader = new HeaderPageletLoader($eventDispatcher, $currencyRoute, $languageRoute, $navigationLoader);
         $header = $headerPageletLoader->load(new Request(), $salesChannelContext);
 
-        static::assertSame($salesChannelContext->getLanguageId(), $header->getActiveLanguage()->getId());
-        static::assertSame($salesChannelContext->getCurrencyId(), $header->getActiveCurrency()->getId());
+        if (!Feature::isActive('v6.8.0.0')) {
```

Comment:
in unit tests the major feature flag is always active

Reply:
Okay, I removed the check from the test :-)